test/mise/ubuntu_with_version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# Optional: Import test library
+source dev-container-features-test-lib
+
+# Definition specific tests
+check "2025.2.0" mise -v
```

Comment:
It should be checked if the version is correct in the `mise -v` output

```suggestion
check "mise version is equal to 2025.2.0" sh -c "mise -v | grep '2025.2.0'"
```